From 8296bc699c398780cca955396fff36787f65dfdf Mon Sep 17 00:00:00 2001
From: Tobias Tefke <tobias.tefke@gmail.com>
Date: Thu, 15 Feb 2018 12:31:34 +0100
Subject: [PATCH 104/249] Add git date and author subcommands to
 submitting-patches guide

Knowing these subcommands is good for developers who start contributing to Lineage by committing code written by others.
This will also reduce the time between commit and merge if the committers need to amend the patch in order to add the correct date and/or author.

Change-Id: Iaed2f87be0f0b05480b43bc7c266d68ff96e58e1
---
 pages/how-tos/submitting_to_gerrit.md | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/pages/how-tos/submitting_to_gerrit.md b/pages/how-tos/submitting_to_gerrit.md
index 61741b1..44ef88d 100644
--- a/pages/how-tos/submitting_to_gerrit.md
+++ b/pages/how-tos/submitting_to_gerrit.md
@@ -235,6 +235,20 @@ See [Git Immersion](http://gitimmersion.com/) for more information.
 
 *  `git status` to see the status of a project.
 
+### `git subcommands`
+
+*  `--date="DATE"` to specify the date of change if necessary. DATE has to be in RFC 2822, ISO 8601 or git internal time format.
+
+   Examples:
+
+     *  RFC 2822: Wed, 17 Jan 2018 17:39:48 +0100
+
+     *  ISO 8601: 2018-01-17T17:39:48+0100
+
+     *  git internal format: 1516210788 +0100
+
+
+* `--author="NAME <EMAIL>"` to name the author if you did not write the patch yourself.
 
 ## Resources
 
-- 
2.17.1

